test/e2e/v1beta1/scripts/gh-actions/run-e2e-experiment.sh

```diff
@@ -44,7 +44,8 @@ fi
 for exp_name in "${EXPERIMENT_FILE_ARRAY[@]}"; do
   echo "Running Experiment from $exp_name file"
   exp_path=$(find ../../../../../examples/v1beta1 -name "${exp_name}.yaml")
-  python run-e2e-experiment.py --experiment-path "${exp_path}" --verbose || (kubectl get pods -n kubeflow && exit 1)
+  python run-e2e-experiment.py --experiment-path "${exp_path}" --namespace kubeflow \
```

Comment:
Do we need to run the test in the kubeflow namespace? I'd like to verify whether the katib controller can operate Experiments deployed out of the namespace in which the katib controller is deployed.

Reply:
It's a good point, maybe we could run our E2Es on `default` namespace.
WDYT @johnugeorge @nagar-ajay ?

Reply:
@andreyvelich  @tenzen-y I tried running random experiment test in the default namespace, but it failed. As per the error logs, I think other namespaces require `katib.kubeflow.org/metrics-collector-injection: enabled` label.
Error log:
```
kubernetes.client.exceptions.ApiException: (400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Audit-Id': '689dd995-d83a-4915-9e69-130043993f14', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': '66727794-bef2-4ca7-a384-6a1a7b8a571d', 'X-Kubernetes-Pf-Prioritylevel-Uid': 'c6d64b7c-508c-4954-b20b-0b391dcc2e85', 'Date': 'Thu, 06 Apr 2023 12:40:50 GMT', 'Content-Length': '326'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"admission webhook \"validator.experiment.katib.kubeflow.org\" denied the request: Cannot create the Experiment \"random\" in namespace \"default\": the namespace lacks label \"katib.kubeflow.org/metrics-collector-injection: enabled\"","code":400}
```

Reply:
Yes, we must add the label to the namespace.

Can you add processes to update the namespace label in `run-e2e-experiment.py` if the namespace passed by `--namespace` doesn't hold the label?

Reply:
sure, will add.

Reply:
Done. Also updated test namespace to `default`.